core/src/main/scala/org/apache/spark/scheduler/JobWaiter.scala

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.spark.scheduler
 
+import sun.misc.{Signal, SignalHandler}
```

Comment:
@hvanhovell sure. I can update my patch to handle the case when ``sun.misc.Signal/SignalHander`` classes don't exit. Thank you!